portal/src/main/webapp/js/src/study-view/component/PieChart.js

```diff
@@ -984,28 +992,28 @@ var PieChart = function(){
             }
 
             if(i % 1 === 0){
-                _tableDiv += "<tr>";
+                _tableDiv.push("<tr>");
             }
 
-            _tableDiv += '<td  uniqueId="'+
+            _tableDiv.push('<td  uniqueId="'+
```

Comment:
here you could use the _.template functionality. See for example how templates from pancancer_study_summary_templates.html are used